app/commands.py

```diff
@@ -258,10 +264,9 @@ def update_jobs_archived_flag(start_date, end_date):
         sql = """update
                     jobs set archived = true
                 where
-                    created_at >= (date :start + time '00:00:00') at time zone 'America/New_York'
-                    at time zone 'UTC'
-                    and created_at < (date :end + time '00:00:00') at time zone 'America/New_York' at time zone 'UTC'"""
-
+                    created_at >= (date :start + time '00:00:00')
+                    and created_at < (date :end + time '00:00:00')
```

Comment:
Just a quick gut check:  these changes are okay in that we don't need to supply a timezone for these DB fields and we're still getting the timestamps back we expect (especially given the flip to UTC), correct?

Reply:
Yes, there is no time on the backend except UTC.  This was an oversight, because it's a weird, one-off pattern.